### ISSUE TITLE
bug: tasker unlicensed when running task

### ISSUE THREAD

**Issue by Dialgatrainer02** [ReVanced/revanced-patches#1408]
### Type

Error at runtime

### Bug description

When running a task with tasker it stopes and errors saying it's unlicensed and to go buy it

### Steps to reproduce

Download tasker from apk mirror
Install and patch with manager with unlock trial patch
Install as root 
Setup and go past authorisation error
Run a task

### Relevant log output

```shell
13.27.55/E Sorry, Tasker can't be used until you purchase it from Google Play. Tap the notification to purchase Tasker from your browser..
13.27.55/TaskerContext AC cancel all: cancel: 13.00.00
13.27.55/TaskerContext AC cancelAll: 13.00.00: already seen
13.27.55/TaskerContext AC cancelAll: 13.00.00: already seen
13.27.55/TaskerContext AC cancelAll: 13.00.00: already seen
13.27.55/TaskerContext AC cancelAll: 13.00.00: already seen
13.27.55/TaskerContext AC cancelAll: 13.00.00: already seen
13.27.55/TaskerContext AC seen 5 repeats, bail
13.27.55/QSTI update: 0
13.27.55/QSTI update: 1
13.27.55/QSTI update: 2
13.27.55/E result: stop task (error)
13.27.55/E Error: 1
13.27.55/MacroEdit action finished exeID 2 action no 5 code 162 status: Err next 5
```

### Additional context

Pixel 6 pro rooted magisk 25200 December security update

Tasker version:6.1.4-beta

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by Scamolor** [comment]
Same issue here. I'm also using Tasker 6.14 beta and I'm getting this issue too. I'm going to check if it affects 6.10.

**Comment by ILoveOpenSourceApplications** [comment]
Issue will be no longer resolved due to removal of the patch due to #2775.
